aiotnb/core.py

```diff
@@ -282,20 +281,29 @@ class LocalAccount:
         .. caution:: Do not share this key. This is the private key and can be used to impersonate you.
     """
 
-    def __init__(self, private_key: SigningKey):
+    def __init__(self, signing_key: str):
```

Comment:
The user shouldn't have to manually init the class at all, the `from_key_file` and `generate` classmethods should be the only way of creating a new LocalAccount instance.

Reply:
What about when they want to init a class from a hex string?